index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="nl">
+<head>
+    <meta charset="UTF-8">
+    <title>COVID-19 Notificatie App Status</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
Nitpick: Do we want to add a meta description for the Googles and Bings of this world?

Reply:
Added a meta description tag in the head